pages/e-lab.tsx

```diff
@@ -124,18 +132,21 @@ export default function AIELab() {
           <p>
             Are you ready for your next step? Great. With our AI
             Entrepreneurship Lab we aim at making the founding of AI{" "}
-            <span className="text-red-500">startups</span>
-            accessible to everyone, including you. You bring in the motivation
-            and dedication, and we connect you and try to make founding as easy
-            as possible for you.
+            <span className="text-red-500">startups </span>
+            accessible to everyone, including you. Bring your motivation and
+            dedication, and we’ll provide the rest, striving to make founding as
+            easy as possible for you.
           </p>
         </div>
-
-        <SnapSlider cards={mentors} />
+        {/*
```

Comment:
Dead code

Reply:
We will leave it in, as the testimonial section using the slider, will be added later on